backend/src/packages/users/user.repository.ts

```diff
@@ -106,15 +112,21 @@ class UserRepository implements Repository {
     return Promise.resolve(null);
   }
 
-  public delete(): ReturnType<Repository['delete']> {
-    return Promise.resolve(true);
+  public async delete(id: number): Promise<boolean> {
+    const updatedRows = await this.userModel
```

Comment:
It can be in plural form

Reply:
@v0ldemar01  do you mean the name of the method delete() should I change it? or do you mean something else?

Reply:
@v0ldemar01  I did all the changes and updated the PR, once you provide me a little more context I can fix this too if needed.